bin/htcondor_make_runtime.sh

```diff
@@ -40,8 +40,9 @@ rm -f $STARTDIR/CRAB3.zip
 rm -f $STARTDIR/WMCore.zip
 
 # make sure there's always a CRAB3.zip to avoid errors in other parts
-touch /tmp/dummyFile
-zip -r $STARTDIR/CRAB3.zip /tmp/dummyFile
+touch $PWD/tmp/dummyFile
```

Comment:
@belforte, I think this is causing the [jenkins build](https://cmssdt.cern.ch/dmwm-jenkins/job/CRABServer_BuildOnRelease/289/consoleFull) to fail with

```
+ touch /build/cmsbld/jenkins/workspace/CRABServer_BuildOnRelease/w/BUILD/slc7_amd64_gcc630/cms/crabtaskworker/py3.220117-90b5464e8b8b9731333847cf197ffbad/CRABServer-py3.220117/tmp/runtime/tmp/dummyFile
touch: cannot touch '/build/cmsbld/jenkins/workspace/CRABServer_BuildOnRelease/w/BUILD/slc7_amd64_gcc630/cms/crabtaskworker/py3.220117-90b5464e8b8b9731333847cf197ffbad/CRABServer-py3.220117/tmp/runtime/tmp/dummyFile': No such file or directory
error: Bad exit status from /build/cmsbld/jenkins/workspace/CRABServer_BuildOnRelease/w/tmp/rpm-tmp.gMaPBU (%build)


RPM build errors:
Macro %rpmbuild_libdir defined but not used within scope
Bad exit status from /build/cmsbld/jenkins/workspace/CRABServer_BuildOnRelease/w/tmp/rpm-tmp.gMaPBU (%build)
```

maybe we can run

```bash
mkdir -t $PWD/tmp
```

before touching the file to make sure that the directory is actually there?

Reply:
will make a new PR :-(